src/generators/codegen.ml

```diff
@@ -936,18 +940,18 @@ module Dump = struct
 							| Var v -> "(" ^ s_access true v.v_read ^ "," ^ s_access false v.v_write ^ ")"
 							| _ -> "")
 						(params f.cf_params);
-					match f.cf_kind with
-						| Var v -> print ":%s;\n" (s_type f.cf_type)
+					(match f.cf_kind with
```

Comment:
Yes the syntax can be a bit tricky. I usually use `begin` and `end` for cases like this to make it safe.